.github/workflows/build.yml

```diff
@@ -15,25 +15,26 @@ jobs:
           17,    # Current Java LTS & minimum supported by Minecraft
         ]
         # and run on both Linux and Windows
-        os: [ubuntu-20.04, windows-2022]
+        os: [ubuntu-22.04, windows-2022]
     runs-on: ${{ matrix.os }}
     steps:
       - name: checkout repository
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
       - name: validate gradle wrapper
         uses: gradle/wrapper-validation-action@v1
       - name: setup jdk ${{ matrix.java }}
-        uses: actions/setup-java@v1
+        uses: actions/setup-java@v3
         with:
           java-version: ${{ matrix.java }}
+          distribution: 'microsoft'
```

Comment:
Why?

Reply:
The v3 version of `actions/setup-java` requires you to specify that.

Reply:
Could you use `temurin` instead of `microsoft`

Reply:
I had it like that because it's the default in the mod template.
I changed it in both workflows.

https://github.com/FabricMC/fabric-example-mod/blob/1.20/.github/workflows/build.yml